pages/repo.page.ts

```diff
@@ -0,0 +1,48 @@
+import { Page } from "@playwright/test";
+import { utils } from "../Utils/utils";
+import { factory } from "../factory";
+import { repoPageSelectors } from "../selectors/repoPage.selectors";
+import { commonData } from "../Data/common.data";
+import { repoPageData } from "../Data/repoPage.data";
+import signinPage from "./signin.page";
+import { operations } from "../enums/operations.enum";
+import { stateEnum } from "../enums/state.enum";
+import { profilePage } from "./profile.page";
+
+export class repoPage
+{
+    private readonly utils : utils
+    
+    constructor(page : Page)
+    {
+        this.utils = factory.initUtils(page);
+    }
+
+    async gotoRepoPage(repoName : string) : Promise<void>
+    {
+        await this.utils.goto(commonData.gitHubUrl + '/' + commonData.accountName + '/' + repoName, repoPageSelectors.codeTab);
+    }
+
+    async gotoCreateRepoPage(signinPage : signinPage, email : string, password : string) : Promise<void>
+    {
+        if(!await this.utils.goto(commonData.gitHubUrl + repoPageData.newRepoUrl, repoPageSelectors.newRepoNameSelector, operations.visible))
```

Comment:
the navigation to new repo page should be by using click(), not goto, this is because the user will use the elements to click on to head to targeted page.

please remove and use the click instead